chain-config/src/lib.rs

```diff
@@ -78,13 +82,51 @@ pub trait ChainConfigContract:
         self.execute_bridge_operation_event(&hash_of_hashes, &config_hash);
     }
 
+    #[endpoint(resumeRegistration)]
+    fn update_registration_status(&self, hash_of_hashes: ManagedBuffer, registration_status: u8) {
+        self.require_setup_complete();
+
+        let status_hash = ManagedBuffer::new_from_bytes(
+            &self
+                .crypto()
+                .sha256(ManagedBuffer::new_from_bytes(&[registration_status]))
+                .to_byte_array(),
+        );
+
+        self.lock_operation_hash(&status_hash, &hash_of_hashes);
+
+        if registration_status != DISABLED && registration_status != ENABLED {
+            self.failed_bridge_operation_event(
+                &hash_of_hashes,
+                &status_hash,
+                &ManagedBuffer::from(INVALID_REGISTRATION_STATUS),
+            );
+
+            self.remove_executed_hash(&hash_of_hashes, &status_hash);
+
+            return;
+        }
+
+        let registration_status_mapper = self.registration_status();
+
+        registration_status_mapper.set(registration_status);
+
+        self.remove_executed_hash(&hash_of_hashes, &status_hash);
+        self.registration_status_update_event(&ManagedBuffer::new_from_bytes(&[
+            registration_status,
+        ]));
```

Comment:
Can you set the buffer like string 'enabled' or 'disabled' instead of '0' or '1'?